jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/PathSpec.java

```diff
@@ -18,67 +18,12 @@
 
 package org.eclipse.jetty.http.pathmap;
 
-/**
- * The base PathSpec, what all other path specs are based on
- */
-public abstract class PathSpec implements Comparable<PathSpec>
+public interface PathSpec extends Comparable<PathSpec>
```

Comment:
While you're editing this file, could you add a header that describes what a PathSpec is.

Reply:
Sure, I'll do it.